app/models/shipit/stack.rb

```diff
@@ -124,10 +124,10 @@ def build_deploy(until_commit, user, env: nil, force: false)
       )
     end
 
-    def trigger_deploy(*args)
-      deploy = build_deploy(*args)
+    def trigger_deploy(until_commit, user, env: nil, force: false, skip_queue: false)
```

Comment:
This is a bit tidier, to avoid duplicating the `build_deploy` signature:

```suggestion
    def trigger_deploy(*args, **kwargs)
      skip_queue = kwargs.delete(:skip_queue)
      deploy = build_deploy(*args, **kwargs)
```